open_spiel/python/mfg/examples/mfg_Proximal_policy_optimization_pytorch.py

```diff
@@ -0,0 +1,448 @@
+import os
+# 
+os.environ["OMP_NUM_THREADS"] = "4" # export OMP_NUM_THREADS=4
```

Comment:
move to main

Reply:
Thank you for your comments Lanctot : ), i restructured some functions are code was introduced into the main.